cpp/ql/lib/semmle/code/cpp/models/interfaces/Throwing.qll

```diff
@@ -10,13 +10,47 @@ import semmle.code.cpp.Function
 import semmle.code.cpp.models.Models
 import semmle.code.cpp.models.interfaces.FunctionInputsAndOutputs
 
+newtype TException =
```

Comment:
Needs QLDoc (and similarly for the IPA branches)

Reply:
Done